### PR TITLE
feat(HIVE-43): remove animation from promo 

### PR DESCRIPTION
**Pull request by maxcbc**
# feat(HIVE-43): remove animation from promo

## What
- remove animation from eventpromo
- update eventpromo props to expect `imageUrl` (string) instead of `images` (string[]) <- **_Breaking change❗_**  
- remove isPaused prop from eventpromo

## Why
- the animation was always paused anyway by default, so more annoying than anything else
- simpler code makes for simpler lives

## Screenshot

Screenshot 2021-02-24 at 11 22 35

### CODE REVIEW

File: src/Image.scss [hunk 1]
```diff
@@ -1,10 +1,41 @@
 @import 'styles/bootstrap';
 
+.img-block {
```
Comment (nickcolley) [inline src/Image.scss:3]: Can you explain why this is needed if we're removing animations?

**Review by nickcolley — APPROVED** [review]
Makes sense to me, make sure to communicate the breaking change in the release notes. 👍🏻

File: src/Image.scss (hunk 1 above)
Comment (maxcbc) [reply src/Image.scss:3]: Have condensed this down to get rid of the div. The css itself is still needed as it makes the 'details' appear in-front of the image.

Comment (nickcolley) [reply src/Image.scss:3]: I think I'm just confused why we need these additional styles when we're just removing the animations, is there also a design change alongside removing animations? Sorry if I'm missing something.

Comment (maxcbc) [reply src/Image.scss:3]: Oh. These were in the ImageContainer.scss which controlled the animation. We no longer need the `<ImageContainer/>` as once the animation bits are removed, but we do need some of the styles (for the z-index, position absolute etc). Hence I've consolidated them into Image.scss and applied them to the anchor tag  in `<Image/>`.

File: src/ImagesContainer.scss [hunk 2]
```diff
@@ -1,53 +0,0 @@
-@import 'styles/bootstrap';
-
-.img-block {
```
Comment (nickcolley) [inline src/ImagesContainer.scss:3]: Ah they're here, thanks for explaining Max :)

**Review by nickcolley — APPROVED** [review]

**Comment by nickcolley** [comment]
Merging on Max's behalf.

**Pull request merged**